libs/data-mapper-v2/src/components/functionList/FunctionListItem.tsx

```diff
@@ -0,0 +1,82 @@
+import { customTokens } from '../../core';
+import type { FunctionData } from '../../models/Function';
+import { getFunctionBrandingForCategory } from '../../utils/Function.Utils';
+import { FunctionIcon } from '../functionIcon/FunctionIcon';
+//import { DMTooltip } from '../tooltip/tooltip';
+import { Button, Caption1, TreeItem, TreeItemLayout, makeStyles, shorthands, tokens } from '@fluentui/react-components';
+import { useStyles } from './styles';
+
+const fnIconSize = '12px';
+
+const useCardStyles = makeStyles({
```

Comment:
Can we please move the styles to a styles.ts file? imo, keeps it cleaner with tsx file just defining the components